Cargo.toml

```diff
@@ -42,6 +42,8 @@ repository = "https://github.com/kanidm/kanidm/"
 argon2 = { version = "0.5.0", features = ["alloc"] }
 async-recursion = "1.0.4"
 async-trait = "^0.1.68"
+axum = {version = "0.6.18", features = ["json", "http2", "macros", "tracing", "headers", "original-uri", "query", "form"]}
+axum-csp = { path="/Users/yaleman/Projects/goatns/axum-csp" }
```

Comment:
Is it okay to use your published crate as well? Like this I was not able to build it.

Reply:
of course - I've released the latest version as 0.0.4 now